gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -287,21 +476,32 @@ cmd void vkGetRenderAreaGranularity(
   map!(u32, VkRect2D) RenderAreas
 }
 
-@internal class
-vkCmdBeginRenderPassArgs {
-  VkSubpassContents        Contents
-  VkRenderPass             RenderPass
-  VkFramebuffer            Framebuffer
-  VkRect2D                 RenderArea
-  map!(u32, VkClearValue)  ClearValues
+@internal class RenderPassBeginInfo {
+  VkRenderPass            RenderPass
+  VkFramebuffer           Framebuffer
+  VkRect2D                RenderArea
+  map!(u32, VkClearValue) ClearValues
+
+  // @extension("VK_KHR_device_group_creation")
   ref!DeviceGroupBeginInfo DeviceGroupBeginInfo
 }
 
-sub void dovkCmdBeginRenderPass(ref!vkCmdBeginRenderPassArgs args) {
+@internal class
+vkCmdBeginRenderPassCommonArgs {
+  ref!RenderPassBeginInfo PRenderPassBeginInfo
+  ref!SubpassBeginInfo    PSubpassBeginInfo
```

Comment:
drop the `P`.

Reply:
Well, I can't because typename is the same with the field name without P.

Reply:
👍